lib/govuk_request_id.rb

```diff
@@ -0,0 +1,13 @@
+class GovukRequestId
+  def self.insert(body)
+    return body unless body.present?
+
+    doc = Nokogiri::HTML::DocumentFragment.parse(body)
```

Comment:
https://github.com/alphagov/govdelivery-email-templates/blob/master/README.md doesn't make mention of any plain text emails, though this doc isn't necessarily up-to-date or complete.